app/views.py

```diff
@@ -17,6 +17,8 @@
 from django.views.generic.base import TemplateView, ContextMixin
 from django.views.generic.detail import DetailView
 from django.views.generic.edit import CreateView, FormView, UpdateView
+from django.views.generic.list import ListView
+import struct
```

Comment:
Put this as the first import in the file and add a newline after it.

Reply:
ah, I ended up not needing struct at all, I'll remove it